flocker/control/_config.py

```diff
@@ -210,8 +210,18 @@ def convert(self):
         if volume:
             config['volume'] = volume
         config['restart_policy'] = self.convert_restart_policy()
+        cpu_shares = self.convert_cpu_shares()
```

Comment:
Well, it impacts `ApplicationMarshaller`, which doesn't directly alter the behaviour of our configuration language, because this class deals only with converting `Application` instances in to some `dict` representation. Should we maybe just move the marshalling class in to `httpapi`?